src/components/policy/policy_regular/include/policy/policy_types.h

```diff
@@ -113,7 +113,13 @@ typedef std::vector<std::string> PermissionsList;
  */
 typedef std::vector<std::string> StringArray;
 
-enum PermitResult { kRpcAllowed = 0, kRpcDisallowed, kRpcUserDisallowed };
+enum PermitResult {
+  kRpcAllowed = 0,
```

Comment:
@AKalinich-Luxoft 
See comment for 'external' implementation

Reply:
@dev-gh fixed in https://github.com/smartdevicelink/sdl_core/pull/1799/commits/92d67279a19c3e4f95011d018c98eb2d66b21629